index.html

```diff
@@ -1816,37 +1816,39 @@ <h3>Lifecycle Details</h3>
 
         <ol>
           <li>
-The <a>verifier</a> defines its policies for accepting
+A <a>verifier</a> defines its own policies for accepting
 <a>verifiable credentials</a> from <a>holders</a> for its supported actions.
           </li>
           <li>
-The <a>issuer</a> issues a <a>verifiable credential</a> to the <a>holder</a>.
+An <a>issuer</a> issues one or more <a>verifiable credentials</a> to 
+a <a>holder</a>.
           </li>
           <li>
-The <a>holder</a> might pass on its <a>verifiable credentials</a> to another
-<a>holder</a>.
+A <a>holder</a> might pass on one or more of its <a>verifiable credentials</a> 
+to another <a>holder</a>.
           </li>
           <li>
-The final <a>holder</a> presents its <a>verifiable credentials</a> to the
-<a>verifier</a> in a <a>verifiable presentation</a>, requesting a supported
+A <a>holder</a> presents one or more of its <a>verifiable credential(s)</a> 
+to a <a>verifier</a> in a <a>verifiable presentation</a>, requesting a supported
 action.
           </li>
           <li>
-The <a>verifier</a> <a>verifies</a> the authenticity of the
-<a>verifiable presentation</a> and <a>verifiable credentials</a>.
+A <a>verifier</a> <a>verifies</a> the authenticity of a <a>verifiable presentation</a> 
+and the <a>verifiable credential(s)</a> contained therein.
           </li>
```

Comment:
The previous step had a holder passing to a verifier. So this step should be talking about the same holder and the same verifier. It makes no sense to talk about any holder or any verifier since they may not have communicated with each other. You are already talking about THE verifiable credentials and not any old VCs, so you are inconsistent. Furthermore the VP is optional so may not be there to be verified. Therefore this text should read


The <a>verifier</a> <a>verifies</a> the authenticity of the presented <a>verifiable credentials</a> 
and the <a>verifiable presentation</a> containing them, if present.